automatic/qbittorrent/update.ps1

```diff
@@ -1,28 +1,36 @@
 import-module au
 import-module "./../../extensions/extensions.psm1"
 
-$releases = 'https://www.fosshub.com/qBittorrent.html'
+$releases = 'http://www.qbittorrent.org/download.php'
+
+function global:au_BeforeUpdate {
+  $download_page = Invoke-WebRequest -uri $Latest.URL32 -UseBasicParsing
+  $url = $download_page.links | ? class -eq "direct-download" | select -expand href
+
+  $client = New-Object System.Net.WebClient
```

Comment:
because iwr is notoriously slow for downloading files, so to save time if there is multiple packages
needing to download files, I'm using .net instead.